app/experimenter/experiments/constants/nimbus.py

```diff
@@ -421,6 +421,20 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_INFREQUENT_MAC_OR_WIN_USER_WITH_5_BOOKMARKS = NimbusTargetingConfig(
+    name="Infrequent Mac or Windows users with 5 bookmarks",
+    slug="infrequent_mac_or_win_users_with_5_bookmarks",
+    # 5 is the new profile bookmark count in Fx 100, a proxy for nothing imported
+    description="Between 1 and 6 days activity in the past 28 days, w/5 bookmarks",
+    targeting="({infrequent_user} && {win_or_mac}) || {sticky}".format(
+        win_or_mac="(os.isWindows || os.isMac)",
+        infrequent_user=TARGETING_INFREQUENT_USER_URIS.targeting,
+        sticky=TARGETING_STICKY,
```

Comment:
Description mentions bookmarks, but targeting doesn't? Does the targeting actually need to be limited to windows and mac and exclude linux?

Reply:
Targeting fixed.  I've asked akash about the linux choice, and will update or comment here once I hear back.

Reply:
PM is happy to have linux, so I've added it in.